pallets/collator-selection/src/lib.rs

```diff
@@ -127,6 +130,12 @@ pub mod pallet {
 		/// This does not take into account the invulnerables.
 		type MaxCandidates: Get<u32>;
 
+		/// Minimum number of candidates that we should have. This is used for disaster recovery.
+		///
+		/// This does not take into account the invulnerables.
```

Comment:
You are saying this count is the number of people + invulnerables?

I think that is over doing it. Invulnerables existing should satisfy the requirement for a minimum number of collators.

Reply:
I agree in theory but invulnerables can be removed all at once (through a permissioned call) I was thinking this would be more of a assume invulnerables would be removed soon thing would be an easier transition, but open to changing

Reply:
Invulnerables should eventually be removed and by then minimum number of collators will be super useful